shell/platform/darwin/macos/framework/Source/FlutterTextInputSemanticsObject.mm

```diff
@@ -0,0 +1,205 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterTextInputSemanticsObject.h"
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterEngine_Internal.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterViewController_Internal.h"
+
+#include "flutter/third_party/accessibility/ax/ax_action_data.h"
+#include "flutter/third_party/accessibility/gfx/geometry/rect_conversions.h"
+#include "flutter/third_party/accessibility/gfx/mac/coordinate_conversion.h"
+
+#pragma mark - FlutterTextFieldCell declaration
```

Comment:
It's generally not useful to `#pragma mark` the separation between declaration and implementation, since Xcode makes those distinct anyway. I would just use a single `#pragma mark - FlutterTextFieldCell` here, and similarly below for the next class. (Or even just `pragma mark -` in both cases.)